src/SettingDrawer/ThemeColor.tsx

```diff
@@ -36,14 +35,13 @@ export interface ThemeColorProps {
 const ThemeColor: React.ForwardRefRenderFunction<
   HTMLDivElement,
   ThemeColorProps
-> = ({ colors, title, value, onChange, formatMessage }, ref) => {
+> = ({ colors, value, onChange, formatMessage }, ref) => {
   const colorList = colors || [];
   if (colorList.length < 1) {
     return null;
   }
   return (
     <div className="theme-color" ref={ref}>
-      <h3 className="theme-color-title">{title}</h3>
```

Comment:
Removing the title from here in favor of using the body title setting, as it is done for the other settings in the drawer.